README.md

```diff
@@ -6,22 +6,21 @@ The purpose of this challenge is to construct complicated queries, learn about j
 - Clone / fork this repo.
 - cd into the repo in your command line `$ cd db-morning-challenge`
 - Install dependencies with `$ npm install`
-- In your browser, go to [ElephantSQL](https://www.elephantsql.com/)
-- Log into ElephantSQL via GitHub
-- Click on 'Create new instance' to create a new database
-- Give your database a name, choose the 'Tiny Turtly' free plan, and select any data center from the list
-- Click on the name of your new new database to see details; you'll need the URL. Copy this to your clipboard!
+- Create an app on Heroku `heroku create app-name-here --region eu` (If you haven't yet installed Heroku CLI see [here](https://devcenter.heroku.com/articles/heroku-cli)
+- Push to Heroku `git push heroku master`
+- Create a new database on Heroku: `heroku addons:create heroku-postgresql:hobby-dev`
```

Comment:
I would add this as an option to creating the database with an alias `heroku addons:create heroku-postgresql:hobby-dev --as USERS_DB`